projects/memapGui/src/main/java/fortiss/gui/listeners/button/DBrowseListener.java

```diff
@@ -24,6 +24,9 @@ public void mouseClicked(MouseEvent e) {
 			String path = file.getPath();
 			DesignerPanel.demandPanel.txtDConsumption.setText(path);
 			d.setConsumptionProfile(path);
-		}
+			
+			// TODO may be the plot should be visible by default.
+			DesignerPanel.demandPanel.plot();
+		}		
```

Comment:
The plot panel is initially hiding in the DemandInputPanel to match the design of other information panels, where the plot panels are hidden due to the number of elements being displayed. Nonetheless, we can create a new issue to make this plot panel permanently visible or initially visible.

Reply:
okay, we leave it as it is, and we need to change that, we open an issue.